docker-compose.unixports.yml

```diff
@@ -0,0 +1,18 @@
+jenkins:
+    user: ${USER_ID}
+
+testintegration:
+    ports:
+        - "14064:14064"
+        - "14063:14063"
+omero:
+    ports:
+        - "4064:4064"
```

Comment:
Just to be clear: we had previously reverted this because of conflicting ports on the same host. Is the underlying assumption here is that devspaces will be managed on isolated hosts/OpenStack tenancies?